runtime/doc/if_lua.txt

```diff
@@ -13,14 +13,17 @@ Lua Interface to Nvim					*lua* *Lua*
 
 Neovim lua interface automatically adjusts `package.path` and `package.cpath` 
 according to effective &runtimepath value.  Adjustment happens after each time 
-'runtimepath' is changed, `package.path` and `package.cpath` are adjusted by 
-prepending directories from 'runtimepath' each suffixed by `/lua` and 
-`?`-containing suffixes from `package.path` and `package.cpath`. I.e. when 
-'runtimepath' option contains `/foo` and `package.path` contains only 
-`./?.lua;./a?d/j/g.nlua;/bar/?.lua` the resulting `package.path` after 
+'runtimepath' is changed. `package.path` is adjusted by simply appending 
```

Comment:
But it is more correct. Adjusting is not triggered by changing &runtimepath, it is triggered by running something like `:lua` after &runtimepath was changed.